src/SSHLibrary/library.py

```diff
@@ -26,6 +26,7 @@
                      StringEntry, TimeEntry)
 from .utils import ConnectionCache, is_string, plural_or_not
 from .version import VERSION
+from robot.utils import is_truthy
```

Comment:
Should be imported via our own `.utils`. May need to be imported there from `robot.utils` if that's not done yet, though.